### PR TITLE
feat: First version

### PR DESCRIPTION
**Pull request by dploeger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# ansible-loki-callback: An Ansible callback plugin that logs to a loki instance
+
+## Requirements
+
+* Current Python3 implementation
```
Comment (timdeluxe) [inline README.md]:
```suggestion
* Python3
```
...why "current"? What does "current" mean? Special version? If so, it should be described more explicit

File: README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+* Ansible
+
+## Installation
+
+Download the repository and install the requirements:
```
Comment (timdeluxe) [inline README.md]:
```suggestion
Download or clone the repository and install the requirements:
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Use the following environment variables to configure the plugin:
+
+* LOKI_URL: URL to the Loki Push API endpoint (https://loki.example.com/api/v1/push)
+* LOKI_USERNAME: Username to authenticate at loki (optional)
+* LOKI_PASSWORD: Password to authenticate at loki (optional)
+* LOKI_DEFAULT_TAGS: A comma separated list of key:value pairs used for every log line (optional)
+* LOKI_ORG_ID: Loki organization id (optional)
+
+Then set `ANSIBLE_CALLBACK_PLUGINS` to the path where you installed the plugin.
```
Comment (timdeluxe) [inline README.md]:
```suggestion
Then set `ANSIBLE_CALLBACK_PLUGINS` to the path where you downloaded or cloned the repository to.
```

File: loki.py [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+import logging_loki
+from ansible.plugins.callback import CallbackBase
+
+DOCUMENTATION = '''
+    callback: loki
+    type: loki
+    short_description: Ansible output logging to loki
+    version_added: 0.1.0
+    description:
+        - This plugin sends Ansible output to loki
```
Comment (timdeluxe) [inline loki.py]: This looks indented wrongly

**Review by timdeluxe — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+* LOKI_DEFAULT_TAGS: A comma separated list of key:value pairs used for every log line (optional)
+* LOKI_ORG_ID: Loki organization id (optional)
+
+Then set `ANSIBLE_CALLBACK_PLUGINS` to the path where you downloaded or cloned the repository to.
+
+## Testing
+
+The example directory contains a test playbook that can be used to test the callback plugin. Run it using
+
+    ANSIBLE_CALLBACK_PLUGINS="${PWD}" ansible-playbook -i example/inventory.yaml example/playbook.yaml -vvvvvv 2>/dev/null
```
Comment (ThomasMinor) [inline README.md]: I like a trailing LF on the last line but I guess MD is a bit more friendly here.

This, BTW, is also my opioion on other txt files.

**Review by ThomasMinor — APPROVED** [review]

**Review by timdeluxe — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (dploeger) [reply README.md]: Added the trailing LF.

**Pull request merged**